ext/gd/tests/gh13082.phpt

```diff
@@ -3,7 +3,7 @@ GH-13082 - imagefontwidth/height unexpectedly throwing an exception on a valid G
 --EXTENSIONS--
 gd
 --SKIPIF--
-<?php if (getenv('TRAVIS')) die('skip Currently fails on Travis'); ?>
+<?php if (pack('i', 0x01020304) !== "\x04\x03\x02\x01") die('skip unsupported platform'); ?>
```

Comment:
It might be better to be clear about "what" the platform does not support.

```suggestion
<?php if (pack('i', 0x01020304) !== "\x04\x03\x02\x01") die('skip big endian is unsupported'); ?>
```

Reply:
Indeed, but it's not only about the endianess, but also `int` must be 32bit. Maybe the latter is pretty much the case everywhere nowadays, but that might change.

Reply:
Ah, I overlooked that. It is impossible to explain this in detail in one sentence.